### ISSUE TITLE
Can no longer edit/view/alter games via right click menu.

### ISSUE THREAD

**Issue by nPHYN1T3** [ValveSoftware/steam-for-linux#9300]
#### Your system information

* Mar 24 2023 @ 17:20:43
* Garuda (Arch)
* No Beta
* System is always current

#### Please describe your issue in as much detail as possible:
I can no longer verify games, set launch parameters or anything else that requires a right click.

Right clicking a game in the library or side menu either locks up and freezes the menu popup or crashes the library view and refreshes/restarts it.
```
systemd-coredump[4010854]: [🡕] Process 4010200 (steamwebhelper) of user 1000 dumped core.

Stack trace of thread 4010200:
#0 0x00007f8685ae8c04 XVisualIDFromVisual (libX11.so.6 + 0x2bc04)
#1 0x00007f8685d058d2 n/a (/home/$USER/.local/share/Steam/ubuntu12_64/libSDL3.so.0 + 0x1058d2)
ELF object binary architecture: AMD x86-64
Mar 25 06:28:41 ------- systemd[1]: systemd-coredump@8-4010853-0.service: Deactivated successfully.
Mar 25 06:28:45 ------- dbus-daemon[5992]: [session uid=1000 pid=5992] Activating service name='org.freedesktop.portal.IBus' requested by ':1.130' (uid=1000 pid=4011153 comm="/home/$USER/.local/share/Steam/ubuntu12_64/steamweb")
Mar 25 06:28:45 ------- org.freedesktop.portal.IBus[4011212]: Not connected to the ibus bus
Mar 25 06:28:45 ------- dbus-daemon[5992]: [session uid=1000 pid=5992] Activated service 'org.freedesktop.portal.IBus' failed: Process org.freedesktop.portal.IBus exited with status 1
```

Right clicking a game in the library list = CAPIJobRequestUserStats - No Connection & no schema

If the attempt hangs all logging dies. If the Library view crashes and restarts the logs show:
CAPIJobRequestUserStats - No Connection & no schema
steamwebhelper.sh[205160]: Runtime for steamwebhelper: defaulting to /home/shed/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[205160]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
BuildCompleteAppOverviewChange: 360 apps

(steam:192338): Gtk-WARNING **: 02:30:29.360: gtk_disable_setlocale() must be called before gtk_init()
RegisterForAppOverview 1: 2ms
RegisterForAppOverview 2: 2ms

#### Steps for reproducing this issue:

1. Have more than one XScreen defined in xorg.conf
2. Run Steam on an XScreen other than 0.0
3. Note the behaviors above because GTK is broken by design for XScreen configs. GTK no longer enumerates XScreens since 3.2(ish) so when Steam calls problematic GTK functions and XScreen 0.1 or 0.2 is returned it just crashes or hangs. This issue was not present until a recent client update. All functions work as intended if Steam is run on XScreen 0.0.

**Comment by nPHYN1T3** [comment]
Interestingly Steam is again functioning. Right clicking only kicks out CAPIJobRequestUserStats - No Connection & no schema and brings up the menu to modify/uninstall/etc correctly again.

**Comment by nPHYN1T3** [comment]
And it's back...

Apr 11 02:03:29 systemd-coredump[3852282]: [🡕] Process 3850517 (steamwebhelper) of user 1000 dumped core.
                                                  
                                                  Stack trace of thread 3850517:
                                                  #0  0x00007f387ee08c04 XVisualIDFromVisual (libX11.so.6 + 0x2bc04)
                                                  #1  0x00007f3874f058d2 n/a ($HOME/.local/share/Steam/ubuntu12_64/libSDL3.so.0 + 0x1058d2)
                                                  ELF object binary architecture: AMD x86-64

**Comment by nPHYN1T3** [comment]
Day after reopening a game won't run, I right click to go to properties and check a few things before I remember that's broken and it hangs for a moment then Steam proceeds to draw garbage again all over my screens and I have to reboot to get it to stop.

**Comment by 88mm** [comment]
I have a similar but not as bad a problem:  The steam client library properties (via right-click on a game) no longer allows me to select entries in drop-down menu.   (If I should open a new issue on this, let me know)

 System information:

* Steam client version:  Build date: Mar 24 2023
* Distribution:  Gentoo stable
* Opted into Steam client beta?: No  (but did try it, see below)
* Have you checked for system updates?: Yes

In the steam client Library (ie game) properties no longer allow me to select entries in drop-down menus. (check boxes still work).
For example I can not change from always keeping a game updated to only updating a game on launch. Or if I choose a specific compatibility tool I can no longer select the proton version in the drop down.

I have tried re-starting the steam client and rebooting my system. I have tested this on several games listed in the steam client library. This is a relatively new issue as I almost always change new games I purchase to update only on launch. 

(I tried the steam Linux beta but that would not even start and I had to delete ~/.steam/steam/package/ beta to get back to the non-beta.)

**Comment by nPHYN1T3** [comment]
My question is what is your GPU/X config? Are you running Steam on a non 0.0 XScreen? If I run Steam on 0.0 it works fine. It's not until Steam is on 0.1 or 0.2 that the behavior shows up.

**Comment by kisak-valve** [comment]
Hello @88mm, your issue is being tracked at #9273.

**Comment by 88mm** [comment]
> My question is what is your GPU/X config? Are you running Steam on a non 0.0 XScreen? (Are you even running X?)
I think I am on 0 ...

$ xdpyinfo
name of display:    :0
version number:    11.0
vendor string:    The X.Org Foundation
vendor release number:    12101008
X.Org version: 21.1.8
...
default screen number:    0
number of screens:    1

screen #0:
  dimensions:    2560x1440 pixels (602x352 millimeters)
  resolution:    108x104 dots per inch
...

**Comment by nPHYN1T3** [comment]
Your issue is not my reported issue. As @kisak-valve said your issue is tracked at https://github.com/ValveSoftware/steam-for-linux/issues/9273

**Comment by nPHYN1T3** [comment]
Right click and various UI elements are still broken...I love how many tickets I have pushing a year with no fix.
